pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -517,14 +522,18 @@ private void uploadSegment(File indexDir, SegmentMetadata segmentMetadata, File
 
       // Update ZK metadata and refresh the segment if necessary
       long newCrc = Long.valueOf(segmentMetadata.getCrc());
+      LOGGER.info("CRC for segment {}, existing: {} new: {}", segmentName, existingCrc, newCrc);
       if (newCrc == existingCrc) {
-        // New segment is the same as the existing one, only update ZK metadata without refresh the segment
+        LOGGER.info("New segment {} is same as existing one, updating ZK metadata without refreshing the segment",
```

Comment:
might want to change the statement to say that new segment crc is same as existing segment crc